portfolio/src/main/webapp/script.js

```diff
@@ -6,15 +6,65 @@ function changeImage() {
 }
 
 /**
-  * Creates and returns an HTML list element containing
-  * a string provided by the the text parameter.
-  * @param {String} text Content of the list element.
-  * @return {<li>} HTML list element with the contents of the text parameter.
+ * ENUM type to specify direction of vote.
+ */
+const Direction = {
+  up: 1,
+  down: -1
+}
```

Comment:
Enum names should be in all caps.

https://google.github.io/styleguide/jsguide.html#jsdoc-enum-and-typedef-comments